controllers/admin/AdminLinkWidgetController.php

```diff
@@ -316,4 +337,17 @@ private function deleteLinkList()
 
         return $success;
     }
+
+    private function getLinkBlockRepository()
+    {
+        if (null === $this->repository) {
+            $this->repository = new LegacyLinkBlockRepository(
```

Comment:
Possibility to use an Adapter with symfony service?

Reply:
this controller will be removed once the whole migration is done